docs/tutorial/mojave-dark-mode-guide.md

```diff
@@ -0,0 +1,46 @@
+# Mojave Dark Mode
+
+In macOS 10.14 Mojave Apple introduced a new [system wide dark mode](https://developer.apple.com/design/human-interface-guidelines/macos/visual-design/dark-mode/)
+for all macOS computers.  By default Electron apps do not automatically adjust their UI and native interfaces
+to the dark mode setting when it's enabled, this is primarily due to Apple's own guidelines saying you **shouldn't**
+use the dark mode native interfaces if the your app's own interfaces don't support dark mode themsleves.
+
+If your app does have a dark mode though, you can make your Electron app follow the system wide dark mode setting.
+
+## Automatically updating the native interfaces
+
+"Native Interfaces" include the file picker, window border, dialogs, context menus and more; basically anything where
+the UI comes from macOS and not your app.  In order to make these interfaces update to dark mode automatically you need
+to set the `NSRequiresAquaSystemAppearance` key in your app's `Info.plist` file to `false`.  E.g.
+
+```xml
+<plist>
+<dict>
+  ...
+  <key>NSRequiresAquaSystemAppearance</key>
+  <false />
+  ...
+</dict>
+</plist>
+```
+
+If you are using [`electron-packager`](https://github.com/electron-userland/electron-packager) or
+[`electron-forge`](https://github.com/electron-userland/electron-forge) you can set the
```

Comment:
Maybe indicate the versions of Electron Packager (>= 12.2.0) / Electron Forge (>= 6) that have support for this flag?